README.md

```diff
@@ -129,13 +129,36 @@ Generates Swift snippets from colors, fonts and layers.
 
 ## How to Install
 
-The instruction from official [tutorial](https://github.com/zeplin/zeplin-extension-documentation/blob/master/tutorial.md#adding-a-local-extension).
+Download and unzip [the latest release](https://github.com/artemnovichkov/zepcode/releases/tag/0.2.0).
```

Comment:
With this logic, you have to update README every release. Let's use just https://github.com/artemnovichkov/zepcode/releases